po/es.po

```diff
@@ -18464,75 +18748,107 @@ msgid ""
 "`JoinHandle` returned from `tokio::spawn` implements `Future` to allow "
 "joining to it."
 msgstr ""
+"Una función asíncrona devuelve `impl Future`. También es posible (aunque no "
+"es habitual) implementar `Future` para tus propios tipos. Por ejemplo, el "
+"`JoinHandle` devuelto por `tokio::spawn` implementa `Future` para permitir "
+"que se una a él."
 
 #: src/async/futures.md:27
 msgid ""
 "The `.await` keyword, applied to a Future, causes the current async function "
 "to pause until that Future is ready, and then evaluates to its output."
 msgstr ""
+"La palabra clave `.await`, aplicada a un futuro, provoca que la función "
+"asíncrona se detenga hasta que dicho futuro esté listo y, a continuación, se "
+"evalúa su salida."
 
 #: src/async/futures.md:32
 msgid ""
 "The `Future` and `Poll` types are implemented exactly as shown; click the "
 "links to show the implementations in the docs."
 msgstr ""
+"Los tipos `Future` y `Polll` se implementan exactamente como se indica. Haz "
+"clic en los enlaces para mostrar las implementaciones en los documentos."
 
 #: src/async/futures.md:35
 msgid ""
 "We will not get to `Pin` and `Context`, as we will focus on writing async "
 "code, rather than building new async primitives. Briefly:"
 msgstr ""
+"No trataremos `Pin` ni `Context`, ya que nos centraremos en escribir código "
+"asíncrono en lugar de compilar nuevos primitivos asíncronos. Brevemente:"
 
 #: src/async/futures.md:38
 msgid ""
 "`Context` allows a Future to schedule itself to be polled again when an "
 "event occurs."
 msgstr ""
+"`Context` permite que un futuro se programe a sí mismo para que se vuelva a "
+"sondear cuando se produzca un evento."
 
 #: src/async/futures.md:41
 msgid ""
 "`Pin` ensures that the Future isn't moved in memory, so that pointers into "
 "that future remain valid. This is required to allow references to remain "
 "valid after an `.await`."
 msgstr ""
+"`Pin` asegura que el futuro no se mueva en la memoria, de forma que los "
+"punteros en ese futuro siguen siendo válidos. Esto es necesario para que las "
+"referencias sigan siendo válidas después de `.await`."
 
 #: src/async/runtimes.md:3
 msgid ""
 "A _runtime_ provides support for performing operations asynchronously (a "
 "_reactor_) and is responsible for executing futures (an _executor_). Rust "
 "does not have a \"built-in\" runtime, but several options are available:"
 msgstr ""
+"Un _tiempo de ejecución_ ofrece asistencia para realizar operaciones de "
```

Comment:
```suggestion
"Un _runtime_ ofrece asistencia para realizar operaciones de "
```

This is a different "run time" from the usual. This does not refer "at runtime" but to something called "runtime" in async rust. I don't think we have a translation for that.